gauntlet/packages/gauntlet-serum-multisig/src/commands/sendRawTx.ts

```diff
@@ -0,0 +1,43 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { logger } from '@chainlink/gauntlet-core/dist/utils'
+import { SolanaCommand, TransactionResponse } from '@chainlink/gauntlet-solana'
+import { Message, Transaction } from '@solana/web3.js'
+import { CONTRACT_LIST } from '@chainlink/gauntlet-solana-contracts'
+
+export default class SendRawTx extends SolanaCommand {
```

Comment:
Hmm, why is this part of `gauntlet-serum-multisig`? 
IMO this base command should be exposed from `guntlet-solana`.

Reply:
Right. Was there for testing purposes only